src/specklepy/transports/server/server.py

```diff
@@ -94,14 +94,19 @@ def __init__(
 
         self.session = requests.Session()
 
+        self.session.headers.update(
```

Comment:
@JR-Morgan is this a fix we forgot to port over to v3-dev?

Reply:
Yes. Here is a list of all the PRs on `main` that aren't on `v3-dev`
![image](https://github.com/user-attachments/assets/9e7839da-8571-4888-8e15-881a115ac00e)